pandas/tests/indexes/test_multi.py

```diff
@@ -632,6 +632,13 @@ def test_from_arrays_index_series_period(self):
 
         tm.assert_index_equal(result, result2)
 
+    def test_from_arrays_different_lengths(self):
+        # GH13599
+        idx1 = [1, 2, 3]
```

Comment:
better yet make sure to check with a 0-len array (check for all 2 cases, idx1 or idx2 is 0-len)